Error with `Window.addToolbar()`

When building the app that has the code `Window.addToolbar()` in it, it fails to build with the following message:
```zsh
/Users/USR1132/flutter/.pub-cache/git/flutter_acrylic-49776d5078b2c482e552fa090a8db948c66b3ec6/macos/Classes/MainFlutterWindowManipulator.swift:391:26: error: 'init()' is only available in macOS 10.13 or newer
        let newToolbar = NSToolbar()
                         ^
        let newToolbar = NSToolbar()
                         ^
    public static func addToolbar() {
                       ^
public class MainFlutterWindowManipulator {
```

This can be fixed by wrapping the code within `addToolbar()` function with this:
```swift
if #available(macOS 10.13, *) {
	//...
}
```

--- a/macos/Classes/MainFlutterWindowManipulator.swift
+++ b/macos/Classes/MainFlutterWindowManipulator.swift
@@ -388,9 +388,11 @@ public class MainFlutterWindowManipulator {
     }
     
     public static func addToolbar() {
-        let newToolbar = NSToolbar()
-        
-        self.mainFlutterWindow?.toolbar = newToolbar
+        if #available(macOS 10.13, *) {
+            let newToolbar = NSToolbar()
+
+            self.mainFlutterWindow?.toolbar = newToolbar
+        }
     }
     
     public static func removeToolbar() {

